### DISCUSSION TITLE
Certificate error on play.verus-lang.org

### DISCUSSION THREAD

**Discussion by eminence** [verus-lang/verus#876] [General]
Hello! 👋 

It seems there is a certificate issue on https://play.verus-lang.org/:

> This server could not prove that it is play.verus-lang.org; its security certificate expired 3 days ago. This may be caused by a misconfiguration or an attacker intercepting your connection. Your computer's clock is currently set to Wednesday, November 1, 2023. Does that look right? If not, you should correct your system's clock and then refresh this page.

**Comment by parno** [comment]
Thanks for the heads up!  It looks like `nginx` didn't reload after the certificate updated, so it was still using the old version.  It should be fixed now.
